### DISCUSSION TITLE
No --exclude-arch for twister

### DISCUSSION THREAD

**Discussion by yashi** [zephyrproject-rtos/zephyr#34960] [Q&A] [answered]
How can I exclude an arch from twister?  Let's say I want to test hello world with all arch except x86, how can I do that?
I know there is `--platform`, `--exclude-platform`, and `--arch`, but I don't see `--exclude-arch`.  Is there any way to specify it?

**Comment by henrikbrixandersen** [comment, marked as answer]
I do not believe you can exclude certain archs from a twister run. Closest thing is to limit the twister run to a set or archs using (any number of) `--arch arm --arch arc`. I am sure a pull request for adding `--exclude-archs` will be welcome.
